server/plugin/api.go

```diff
@@ -143,10 +143,11 @@ func (p *MatterpollPlugin) handlePostActionIntegrationRequest(handler postAction
 			p.API.LogWarn("failed to handle PostActionIntegrationRequest", "error", err.Error())
 		}
 
-		response := &model.PostActionIntegrationResponse{}
 		if msg != nil {
-			response.EphemeralText = p.LocalizeDefaultMessage(userLocalizer, msg)
+			p.SendEphemeralPost(request.ChannelId, request.UserId, p.LocalizeDefaultMessage(userLocalizer, msg))
```

Comment:
This is the important change

Reply:
Although I don't have any negative opinions for this changes, I don't understand importances for this fixes. Could you tell me any hints?

Reply:
Currently ephemeral messages are send `model.PostActionIntegrationResponse.EphemeralText`. The ephemeral message is posted as the user and the user icon and user name are overwritten to let it look like a Matterpoll message.

With this change we just use the bot account to create the message. This has the following advantages:
- We don't rely on overwriting the user icon
- The user icon is served (and cached) by the server, not by Matterpoll
- Clicking on the icon doesn't brings up the own users pop-up.

These aren't great advantages, but I think it's good be consistent and use the bot account everywhere.

Reply:
Thanks you for clarifying! I understand the importance of this change, and I agree with your thought :+1:. I also think that using bot account is better.